hazelcast/src/main/java/com/hazelcast/util/scheduler/EntryTaskScheduler.java

```diff
@@ -47,5 +48,14 @@
 
     void cancelAll();
 
+    /**
```

Comment:
My approach doesn't introduce an extra method on a EntryTaskScheduler which has nothing to do with partitions.